### PR TITLE
added Authorization and Heading.tsx

### PR DESCRIPTION
**Pull request by Adouleur**

### CODE REVIEW

**Comment by MiradorOne** [comment]
image

File: app/pages/ForgotPage.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Heading from "~/components/Heading";
+import FormInput from "~/components/Authorization/FormInput";
+import { Button } from "~/components/Button";
+import { LeftArrowSvg } from "~/components/IconComponents";
+import { Link } from "@remix-run/react";
+
+const ForgotPage = () => {
+  const onChangeEmailHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    console.log(e.target.value);
```
Comment (MiradorOne) [inline app/pages/ForgotPage.tsx]: remove console.log

File: app/pages/LoginPage.tsx [hunk 2]
```diff
@@ -1,6 +1,53 @@
...
+import GoogleAuth from "~/components/Authorization/GoogleAuth";
+import Heading from "~/components/Heading";
 
 const LoginPage = () => {
-  return <div>Login Page</div>;
+  const [errorMessages, setErrorMessages] = useState({});
+  const [login, setLogin] = useState("");
+  const [password, setPassword] = useState("");
+  const onChangeLoginHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    console.log(e.target.value);
```
Comment (MiradorOne) [inline app/pages/LoginPage.tsx]: same here

File: app/pages/SignUpPage.tsx [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+const SignUpPage = () => {
+  const [errorMessages, setErrorMessages] = useState({});
+  const [login, setLogin] = useState("");
+  const [password, setPassword] = useState("");
+  const [confirmPassword, setConfirmPassword] = useState("");
+  const [isPasswordsSame, setIsThePasswordsSame] = useState(true);
+  const [isPasswordShort, setIsPasswordShort] = useState(false);
+
+  const onChangeNameHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    console.log(e.target.value);
```
Comment (MiradorOne) [inline app/pages/SignUpPage.tsx]: same

**Review by MiradorOne — CHANGES_REQUESTED** [review]
remove unnessesary .logs

**Comment by Adouleur** [comment]
Done! Please check

**Pull request merged**